deliver/lib/deliver/upload_metadata.rb

```diff
@@ -111,7 +111,7 @@ def upload(options)
       end
 
       # Needed for to filter out release notes from being sent up
-      number_of_versions = app.get_app_store_versions(filter: { platform: platform }, limit: 2).size
+      number_of_versions = app.get_app_store_versions_count(filter: { platform: platform }, limit: 2)
```

Comment:
```suggestion
      number_of_versions  = Spaceship::ConnectAPI.get_app_store_versions(app_id: app.id, filter: { platform: platform }, limit: 2).count
```

What if we change to this? 😇

Reply:
👍